### PR TITLE
feat(DDI-1176): Added ondimiss prop in the notification banner component

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Added onDismiss property in notification banner component

### CODE REVIEW

File: libs/web-components/src/components/notification/Notification.spec.ts [hunk 1]
```diff
@@ -21,4 +21,16 @@ describe('GoANotificationComponent', () => {
...
     mock.mockRestore();
   });
+
+  it("Event triggered on notification banner dismiss", async () => {
+    const onDismiss = jest.fn();
+    const { container } = render(GoANotification, { type: "information" });
+    const notificationBanner = container.querySelector(".information");
+    notificationBanner.addEventListener("_dismiss", onDismiss);
+
+    await fireEvent(notificationBanner, new CustomEvent("_dismiss"));
```
[the remark this answers is not in the document]
Comment (Radha-Alberta) [reply libs/web-components/src/components/notification/Notification.spec.ts]: I have updated the test

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**